vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SlideManagerTest.java

```diff
@@ -0,0 +1,112 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.data.SlideRepository;
+import edu.asu.diging.vspace.core.exception.SlideDoesNotExistException;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+
+public class SlideManagerTest {
+
+	@Mock
+	private SequenceRepository sequenceRepo;
+
+	@Mock
+	private SlideRepository slideRepo;
+
+	@InjectMocks
+	private SlideManager slideManagerToTest = new SlideManager();
+
+	// setting common used variables and Objects
+	String slideId, moduleId, sequenceId, slideIdNotInSequence;
+	List<ISlide> slidesList = Arrays.asList(new Slide());
+	Sequence sequenceObj;

```

Comment:
should all be private. not as important here as in the main code, but still a good practice.

Reply:
Okay, I will change it.